app/models/vm/operations.rb

```diff
@@ -41,10 +41,13 @@ module Vm::Operations
     end
 
     supports :collect_running_processes do
-      check = validate_collect_running_processes
-      unless check[:message].nil?
-        unsupported_reason_add(:collect_running_processes, _(check[:message]))
-      end
+      reason   = N_('VM Process collection is only available for Windows VMs.') unless ['windows'].include?(platform)
```

Comment:
`['windows'].include?(platform)`

This is curious, wonder why it isn't just `platform == 'windows'`  I see you just moved this from existing code though so many we can refactor in a follow-up